design/multi-checkout.md

```diff
@@ -58,26 +58,30 @@ If it's used again, it's OK to re-fetch the same data.
 ### Behavior changes in multi-checkout mode
 
 #### Checkout directory
-By default, all repositories will be checked out as if they've been `git clone`d into the `Pipeline.Workspace` directory.
+By default, all repositories will be checked out as if they've been `git clone`d into the `Build.SourcesDirectory` directory.
 Git uses the last part of the path, minus any trailing slashes, spaces, and the string `.git`.
 (We have YAML syntax for selecting a different path, and if the user specifies that, we'll use their name.)
-This changes the default directory for the `self` repo: it would be `$(Pipeline.Workspace)/s` if it were the only `checkout` step.
+This changes the default directory for the `self` repo: it would be `$(Pipeline.Workspace)/s` if it were the only `checkout` step, but now it will be `$(Pipeline.Workspace)/s/<reponame>`.
 
-The pipeline workspace directory has three well-known subdirectories:
+The pipeline workspace directory has four well-known subdirectories:
+- `s/` for sources
 - `a/` for artifacts
 - `b/` for generated binaries
 - `testResults/` for test results files
 
-We'll recommend not to use these as checkout paths, but we won't attempt to block them.
-Likewise, a customer could choose to checkout a non-`self` repo into the `s` directory, and we won't block that.
+We'll recommend not to use any of these as checkout paths, but we won't attempt to block them.
```

Comment:
This statement is confusing to me. We are saying use `s/` to checkout correct?

Reply:
Well I'm 6 minutes too late lol

Reply:
Haha, never too late. The net of it is: in multicheckout, we'll always create a `s/` directory and call it `Build.SourcesDirectory`. By default, code will be checked out into directories under it. The user can override our default by saying `path` on the checkout and put their source wherever they want.